cmd/argocd/commands/app.go

```diff
@@ -1456,7 +1497,14 @@ func NewApplicationSyncCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 				errors.CheckError(err)
 
 				if !async {
-					app, err := waitOnApplicationStatus(acdClient, appName, timeout, false, false, true, false, selectedResources)
+					appStatusOpts := applicationStatusOpts{
+						client:            acdClient,
+						appName:           appName,
+						timeout:           timeout,
+						watch:             watchOpts{operation: true},
```

Comment:
Since we are waiting for the application to sync is it necessary to enable `sync` in the watchOpts?